install

```diff
@@ -9,6 +9,15 @@ update_config=2
 binary_arch=
 allow_legacy=
 shells="bash zsh fish"
+config_base=~/.
+fish_dir=~/.config/fish
```

Comment:
No slash at the end. Actually, I don't like requiring `$fish_dir` to have a trailing slash, and I would much prefer `$fish_dir/functions` over `${fish_dir}functions` as it's cleaner and less bug-prone. I can see that it's necessary for `$config_base`, so we have either `~/.fzf.bash` or `~/.config/fzf/fzf.bash` but we don't have to do the same for `$fish_dir`. Maybe we can rename `$config_base` to `$config_prefix`, to make the difference more explicit.

Reply:
Done